src/sdg/organisaties/management/commands/send_notification_mail.py

```diff
@@ -0,0 +1,64 @@
+import datetime
+
+from django.conf import settings
+from django.core.mail import send_mail
+from django.core.management import BaseCommand
+from django.utils.html import strip_tags
+from django.utils.timezone import now
+
+from compat import render_to_string
+from dateutil.relativedelta import relativedelta
+
+from sdg.accounts.models import User
+from sdg.core.models.config import SiteConfiguration
+from sdg.producten.models.product import ProductVersie
+
+
+class Command(BaseCommand):
+    help = f"Send product update notification emails from last ({settings.SDG_MAIL_TEXT_CHANGES_EVERY_DAYS}) days to subscribed users."
```

Comment:
Can you add another argument to this command?

``--user=<email or username>`` which indicates the mail needs to be sent to a single user instead of all users.